[2025-02-14T23:30:03.235+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T23:30:03.273+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-14T23:29:21.176656+00:00 [queued]>
[2025-02-14T23:30:03.296+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-14T23:29:21.176656+00:00 [queued]>
[2025-02-14T23:30:03.296+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-14T23:30:03.319+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): assign_permissions_dag> on 2025-02-14 23:29:21.176656+00:00
[2025-02-14T23:30:03.326+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1836) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T23:30:03.329+0000] {standard_task_runner.py:72} INFO - Started process 1837 to run task
[2025-02-14T23:30:03.330+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'manage_***_roles', 'assign_permissions_dag', 'manual__2025-02-14T23:29:21.176656+00:00', '--job-id', '366', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp3wn_8l8c']
[2025-02-14T23:30:03.332+0000] {standard_task_runner.py:105} INFO - Job 366: Subtask assign_permissions_dag
[2025-02-14T23:30:03.407+0000] {task_command.py:467} INFO - Running <TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-14T23:29:21.176656+00:00 [running]> on host 71404c6b5320
[2025-02-14T23:30:03.632+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='manage_***_roles' AIRFLOW_CTX_TASK_ID='assign_permissions_dag' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T23:29:21.176656+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T23:29:21.176656+00:00'
[2025-02-14T23:30:03.634+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-14T23:30:06.698+0000] {dag.py:225} INFO - Assigned permissions to team-hello-world for DAG hello_world_dag
[2025-02-14T23:30:09.661+0000] {dag.py:225} INFO - Assigned permissions to team-hello-world for DAG hello_world_dag
[2025-02-14T23:30:12.867+0000] {dag.py:225} INFO - Assigned permissions to team-hello-world for DAG hello_world_dag
[2025-02-14T23:30:12.869+0000] {dag.py:205} INFO - Permission for DAG manage_***_roles is already correctly assigned to data-team
[2025-02-14T23:30:12.869+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-14T23:30:12.877+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-14T23:30:12.877+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=manage_***_roles, task_id=assign_permissions_dag, run_id=manual__2025-02-14T23:29:21.176656+00:00, execution_date=20250214T232921, start_date=20250214T233003, end_date=20250214T233012
[2025-02-14T23:30:12.889+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T23:30:12.906+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-14T23:30:12.907+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
